meetings/2023-11-01.md

```diff
@@ -43,34 +50,137 @@
+* SC: https://github.com/virginiaBalseiro/solid-ecosystem-monitor/ now includes code using W3C API (see `node-w3capi` directory) towards helping with the election process, in particular determining eligible voters. This data will be passed to W3C Staff for the questionnaire/verifying results. Will add more documentation. Run `npm install` and `node index.js`. Try not to generate too often, W3C and Cloudflare seems to rate limit (429) or other issues (520).
 * SC: There is the possibility that some organisations may not respond to the call on having one representative for putting their ballot, and more than one participant with the same affiliation may vote. To work around this, simple solution: take the vote from the participant that has their name earlier in alphabetical order.
-
-
-### Add section on decisions and processing-pull-requests
-URL: https://github.com/solid/specification/pull/590
-
-* SC: Further details decisions and processing PRs based on https://www.w3.org/2023/Process-20230612/#correction-classes
+* ...: each org will have one representative that will vote
+* ...: W3C staff will assist with the processing of results
+* ...: For orgs that don't respond, identifying which representative will vote, we will provide an ordered list of participants. In that case, if both A and B vote on behalf of that org, then A's vote will be counted
+* ...: Code can be reviewed; we need to ensure that there are no bugs
+* ...: orgs will be ordered by number of participants; those with more than one will be mentioned on the mailing list so that a representative can be named
+* eP: another option may be to work with W3C to enforce certain authorization rules within their system. There are ways to accomplish this, but it will likely be a lot of manual work. Personally, I don't see a problem with this type of enforcement
+* ...: I don't want to bite off to much with this process
+* SC: we're using as much as we can, given what W3C offers
+* ...: The questionnaire is limited to CG participants. The W3C is curious to see how this will work with a CG
+* ...: Other CGs have a similar process
+* ...: The W3C will announce the winners on the mailing list
+* ...: We use W3C infrastructure to take advantage of existing W3C accounts
+* AC: I was wondering if you have a timeframe in mind for when we will initiate voting.
+* SC: once we have eligible voters ready. Let's say within 1 week to announce to mailing list
+* AC: I find it helpful if we have dates, we've been talking about the vote for a while now.
+* SC: I think this is the same issue as we had with spec releases.
+* AC: From my own experience I'm really confused when it comes to the timeframe. A lot of folks would love to see it happen. Many folks already nominated themselves. If we have some dates it would be a good motivation to make it happen.
```

Comment:
```suggestion
* AC: From my own experience, I'm really confused when it comes to the timeframe. A lot of folks would love to see it happen. Many folks already nominated themselves. If we have some dates, it would be a good motivation to make it happen.
```